src/Symfony/Component/Process/PhpProcess.php

```diff
@@ -49,6 +50,14 @@ public function __construct(string $script, string $cwd = null, array $env = nul
         parent::__construct($php, $cwd, $env, $script, $timeout);
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public static function fromShellCommandline(string $command, string $cwd = null, array $env = null, $input = null, ?float $timeout = 60)
+    {
+        throw new LogicException(sprintf('The "%s" cannot be called when using "%s"', __METHOD__, self::class));
```

Comment:
missing final dot :)

Reply:
missing work "method" also, and brackets

Reply:
Improved 🙂